rollup.config.js

```diff
@@ -200,22 +197,33 @@ function buildTarget(buildType, moduleFormat) {
         name: 'pc'
     };
 
+    const sdkVersion = {
+        _CURRENT_SDK_VERSION: version,
+        _CURRENT_SDK_REVISION: revision
+    };
+
     const jsccOptions = {
         debug: {
             values: {
-                _DEBUG: 1,
-                _PROFILER: 1
+                ...sdkVersion,
+                ...{
+                    _DEBUG: 1,
+                    _PROFILER: 1
+                }
```

Comment:
This looks a bit verbose/strange, why not just add the properties directly? Saves two lines and indentation.